use-cases/inferencing/cost-optimization/gcsfuse/AchievingFasterPodStartup.md

```diff
@@ -0,0 +1,319 @@
+models are small or large, or you’re using entry-level or premium accelerators,
+we recommend that you use the following optimizations:
+
+1. **Create Cloud Storage buckets in the workload's region and enable
+   hierarchical namespace:**
+
+To improve read performance of your LLM, we recommend that you enable
+[hierarchical namespace](https://cloud.google.com/storage/docs/hns-overview#key_features)
+in your Cloud Storage bucket. Below is an example of how to create a
+hierarchical bucket in region _us-central1_ using the _gcloud_ command.
+
+```shell
+gcloud storage buckets create gs://BUCKET_NAME \
+  --location=us-central1 \
+  --uniform-bucket-level-access \
+  --enable-hierarchical-namespace
+```
+
+**Note:** If you use multi-regional buckets, you can improve your costs and
+performance by enabling Anywhere Cache, as discussed in the next section.
+
+2. **Enable parallel downloads and caching in Cloud Storage FUSE:**
+
+The yaml snippets highlighted below show how to configure Cloud Storage FUSE to
+download multiple files concurrently and to cache the downloaded data
+indefinitely. For more details, see
+[Optimize Cloud Storage FUSE CSI driver for GKE performance](https://cloud.google.com/kubernetes-engine/docs/how-to/cloud-storage-fuse-csi-driver-perf#parallel-download)
+and the full content of the below snippet below in
+[_model-deployment-a100-tuned.yaml_](https://github.com/GoogleCloudPlatform/accelerated-platforms/blob/main/use-cases/inferencing/cost-optimization/gcsfuse/manifests/model-deployment-tuned-a100-dws.yaml)
+file.
+
+```
+apiVersion: apps/v1
+kind: Deployment
+metadata:
+  name: vllm-openai-gcs-tuned-llama33-70b-a100
+spec:
+  ...
+  template:
+      annotations:
+        gke-gcsfuse/volumes: "true"
+        gke-gcsfuse/cpu-limit: "0"
+        gke-gcsfuse/memory-limit: "0"
+        gke-gcsfuse/ephemeral-storage-limit: "0"
+      ...
+      - name: inference-server
+        ...
+        volumeMounts:
+        - name: gcsfuse
+          mountPath: /gcs/${MODEL_NAME}/${MODEL_VERSION}/
+          readOnly: true
+      volumes:
+      - name: gcsfuse
+        csi:
+          driver: gcsfuse.csi.storage.gke.io
+          volumeAttributes:
+            bucketName: ${GCS_BUCKET}
+            mountOptions: "metadata-cache:ttl-secs:-1,metadata-cache:stat-cache-max-size-mb:-1,metadata-cache:type-cache-max-size-mb:-1,metadata-cache:negative-ttl-secs:0,file-cache:max-size-mb:-1,file-cache:cache-file-for-range-read:true,file-cache:enable-parallel-downloads:true,implicit-dirs,only-dir:${MODEL_NAME}/${MODEL_VERSION}/"
+            skipCSIBucketAccessCheck: "true"
+       ...
+```
+
+**Note**: Smart _mountOptions_ defaults are set from GKE version v1.33. So, if
+you are deploying your workload in G2, A2, A3, A4, A4X, ct5l, ct5lp, ct5p, and
+ct6e machine families, all you need to do is to configure _only-dir_ parameter
+
+3. **Prefer RAM disk for Cloud Storage FUSE caching:**
```

Comment:
Same comment as above: instead of numbered bold text, can it be a smaller heading?